### ISSUE TITLE
RFE: calculate percentage of cloud readiness for assignments wiki

### ISSUE THREAD

**Issue by sadsfae** [redhat-performance/quads#254]
This is an RFE using some cheese math to display a %complete of how far along QUADS is with moving, provisioning, validating hosts.

First, we take the number of systems intended for the assignment (this is already calculated in assignments wiki)

```
quads-cli --cloud-only cloud12 | wc -l
309
```

Next, we call `quads-cli --dry-run` and parse out only those slated to move to the new cloud

```
quads-cli --move-hosts --dry-run | grep "to cloud12" | wc -l
148
```

Finally, we figure out the percentage those remaining systems make up as a total of the final, allotted amount

```
309 / 148 = 0.478 = 47% of hosts allocated.
```

Lastly in https://github.com/redhat-performance/quads/blob/master/quads/tools/create_input_assignments.py we display this value next to the **Validating** area here:

![Screenshot_2019-05-29_12-26-39](https://user-images.githubusercontent.com/3940978/58553811-ca327180-820d-11e9-8c12-0104eae15ece.png)

**Comment by kambiz-aghaiepour** [comment]
Can we have an animated badfish that is in the process of eating a status bar from left to right? And the percentage of readiness corresponds with how much of the status bar the badfish has chomped?

**Comment by grafuls** [comment]
This is already resolved on 1.1.
